RELEASENOTES-1.2.docu

```diff
@@ -165,7 +165,11 @@
       return value in an intermediate local variable.</add-note></build-id>
   <build-id value="next"><add-note> If a bank declared with
       <tt>use_io_memory=false</tt> in a DML 1.4 file is imported from
-      a DML 1.2 device, then the device will now honor overrides of
-      the <tt>transaction_access</tt> method. This simplifies future
-      transition to the Simics 7 API.</add-note></build-id>
+      a DML 1.2 device, using the `dml12-compatibility.dml` library,
+      then a call to the <tt>transaction_access</tt> method will now
+      trigger a proper bank access. Also, if a bank uses the
+      `adml12_compat_io_memory_access` template, then overrides of
```

Comment:
Use `<tt> </tt>` instead of `` ` ` ``, and `s/adml12_compat_io_memory_access/dml12_compat_io_memory_access`

Reply:
Done